src/jobs/time_entries_sync_job.ts

```diff
@@ -64,7 +63,7 @@ export class TimeEntriesSyncJob extends SyncJob {
       const serviceTimeEntriesWrapper = new ServiceTimeEntriesWrapper(
         serviceDefinition,
         syncedService,
-        await syncedService.getTimeEntries(start, now),
+        await syncedService.getTimeEntries(start, now), //TODO chytat vyjimku pri spatnem dotazu a nastavit result jako "Failed"
```

Comment:
Zpracovat TODO

Reply:
To todo tam už nemělo být, bylo vyřešeno v metodě bez házení vyjímek. Odstraněno